.github/workflows/update-changelog.yml

```diff
@@ -0,0 +1,43 @@
+name: "Update Changelog"
+
+on:
+  workflow_call:
+  release:
+    types: [released]
```

Comment:
I think we don't need `release` here, correct? Because we don't want this workflow to run on this repo. Only in a re-use of a different repo. You define this on the calling repo I think.

Reply:
Correct, I've removed the trigger in https://github.com/laravel/.github/pull/3/commits/7df47f5b30760ef469e2d021e48cf50f0520ea04.